src/ontology/cl-edit.owl

```diff
@@ -32177,6 +32180,27 @@ AnnotationAssertion(rdfs:label obo:CL_4042011 "subpial interlaminar astrocyte"@e
 SubClassOf(obo:CL_4042011 obo:CL_4042009)
 SubClassOf(obo:CL_4042011 ObjectSomeValuesFrom(obo:RO_0002170 obo:UBERON_0002361))
 
+# Class: obo:CL_4042012 (L2 marginal neuron)
+
+AnnotationAssertion(obo:IAO_0000028 obo:CL_4042012 "L2MN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29311847") Annotation(oboInOwl:hasDbXref "PMID:32108571") obo:IAO_0000115 obo:CL_4042012 "An L2 intratelencephalic projecting glutamatergic neuron with a soma on the L1-L2 border. This neuron type has small apical dendrites projecting to L1.")
+AnnotationAssertion(terms:contributor obo:CL_4042012 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042012 "2024-04-23T09:31:39Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29311847") Annotation(oboInOwl:hasDbXref "PMID:32108571") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasExactSynonym obo:CL_4042012 "L2MN")
```

Comment:
As it is an acronym, it should be a related synonym.

Reply:
All done